src/java.base/share/classes/java/lang/System.java

```diff
@@ -794,6 +782,16 @@ public static native void arraycopy(Object src,  int  srcPos,
      * </tbody>
      * </table>
      * <p>
+     * The {@code java.specification.maintenance.version} property is
+     * defined if the specification implemented by this runtime at the
+     * time of its construction had undergone a <a
+     * href="https://jcp.org/en/procedures/jcp2#3.6.4">maintenance
+     * release</a>. When defined, its value identifies that
+     * maintenance release. To indicate the first maintenance release
```

Comment:
The final sentence can be shortened, and looking at it now the semicolon should just be a comma:
```
     * maintenance release. To indicate the first maintenance release
     * this property will have the value {@code "1"}, to indicate the
     * second maintenance release it will have the value {@code "2"},
     * and so on.
```
Otherwise, this looks good to me.